### DISCUSSION TITLE
Can 3.6.2 and earlier clients see new Genres?

### DISCUSSION THREAD

**Discussion by chrisrimple** [jamulussoftware/jamulus#1340] [Technical Support & Help] [answered]
Jamulus users are asking on Facebook if people running pre-3.7.0 clients can see new genres like Choral/Barbershop?

If not, then it seems like we should change future clients to request a current list of genres from a central server on startup, rather than having the list hard-coded in the client.

**Comment by hoffie** [comment, marked as answer]
> Jamulus users are asking on Facebook if people running pre-3.7.0 clients can see new genres like Choral/Barbershop?

I think there is no way to select them from the list. However, they should work by using a custom central server field.

> If not, then it seems like we should change future clients to request a current list of genres from a central server on startup, rather than having the list hard-coded in the client.

So, a directory of directory servers? ;)
Sounds like it would have benefits, yes (not having thought about if or how to implement it though).

**Reply by gene96817** [reply]
> I think there is no way to select them from the list. However, they should work by using a custom central server field.

How?

**Reply by hoffie** [reply]
Use Settings > Custom Central Server Adress to specify the address directly. You can find the addresses here: https://jamulus.io/wiki/Central-Servers

**Comment by ann0see** [comment]
Since this is a discussion going off topic should be acceptable:

I already thought of not hard coding the public directory servers in the client. (Personally, I use a custom client for my choir which only has my own central server as set in the code.)

I think it would be great to load the server IPs etc. from a file during runtime.

You'd be able to edit a config file which contains the official central servers as default, modify it to add multiple custom central servers,...

**Reply by jujudusud** [reply]
Really good Idea !
Is it possible to provide a graphical interface in order to be able to modify the configuration file ?
Not usefull for me but for users uses to this kind or configuration windows.

**Reply by ann0see** [reply]
Probably yes, but I'd not (yet) focus on that.
